tests/service/env_setup.yml

```diff
@@ -61,13 +61,3 @@
   ipahostgroup:
     ipaadmin_password: SomeADMINpassword
     name: hostgroup02
```

Comment:
This change is unrelated.

Reply:
This change is in a TEMP commit, so service tests pass.